lib/src/model/podcast.dart

```diff
@@ -295,7 +312,37 @@ class Podcast {
         season: item.itunes?.season,
         episode: item.itunes?.episode,
         chapters: chapters,
+        value: value,
       ));
     });
   }
+
+  static Value _loadValue(RssPodcastIndexValue rssValue) {
+    if (rssValue == null || rssValue.type == null || rssValue.method == null) {
+      return null;
+    }
+
+    List<ValueRecipient> recipients = rssValue.recipients
+      .where((r) => r != null)
+      .where((r) => r.type != null && r.address != null && r.split != null)
```

Comment:
```suggestion
      .where((r) => r?.type != null && r?.address != null && r?.split != null)
```

The above suggestion allows us to remove the previous line